lib/parallel_tests/grouper.rb

```diff
@@ -48,8 +48,15 @@ def in_even_groups_by_size(items, num_groups, options = {})
 
       private
 
+      def specified_groups(options)
+        groups = options[:specify_groups]
+        return groups if groups != '-'
+
+        $stdin.gets
```

Comment:
I think this would be better to be `$stdin.read.chomp` so that stdin is read completely. If you keep `gets` you should `gets(chomp: true)` or else your split will contain a newline in the case of `echo 'foo' | parallel_tests --specify-groups -` and not do what you want